app/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <declare-styleable name="NestedScrollableHost">
+        <attr name="scrollDirection" format="enum">
```

Comment:
yea, but maybe we can keep it? it may come in handy some time, this is the code that searches for the next viewpager parent with the same main scrolldirection.